src/main/java/teammates/ui/controller/Action.java

```diff
@@ -367,21 +367,19 @@ public ActionResult executeAndPostProcess() throws EntityDoesNotExistException {
         return response;
     }
 
+    /**
+     * Adds the list of status messages generated after performing an action to session.
+     * @param response ActionResult generated from performing a certain Action
```

Comment:
Just "@param response ActionResult" ?
There should not be any requirement that is it generated by executing an Action, although that is how it is created.